### DISCUSSION TITLE
Is it possible to backup the databases from the secondary replica?

### DISCUSSION THREAD

**Discussion by andyschweizer** [olahallengren/sql-server-maintenance-solution#366] [Questions & Answers]
We want to backup the databases in our Alwas On Availability Group from the secondary replica.
We want to reduce the network traffic on the primary server.

Is it possible to change the default settings?

Thank You!

> https://ola.hallengren.com/frequently-asked-questions.html

> Does the SQL Server Maintenance Solution support Always On Availability Groups?
> 
> The SQL Server Maintenance Solution supports AlwaysOn Availability Groups.
> Copy-only full backups and transaction log backups are performed on the preferred replica. Full backups (non copy-only) and differential backups are performed on the primary replica.
> 
> Integrity checks are performed on all replicas. Index and statistics maintenance is performed on the primary replica.

**Comment by CarlosGzz-SQL** [comment]
SQL Server only allows you to run full backups on the primary replica, you can do a copy only on secondaries. refer to the documentation.

https://docs.microsoft.com/en-us/sql/database-engine/availability-groups/windows/active-secondaries-backup-on-secondary-replicas-always-on-availability-groups?view=sql-server-ver15

**Comment by olahallengren** [comment]
I have some information about this, in the Frequently Asked Questions.

"How should I configure DatabaseBackup to back up an Availability Group?"

https://ola.hallengren.com/frequently-asked-questions.html
